src/index.html

```diff
@@ -11,12 +11,72 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="logo"
+        href="#"
+      >
+        <img
+          src="/src/images/logo.png"
+          alt="moyo-logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#apple"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link">Photo</a>
+          </li>
+          <li class="nav__item last-link">
```

Comment:
Class names represent the meaning of the content, not the styles or tag names. The class 'last-link' seems to describe the style or position rather than the content. Consider using a name that represents the content or purpose of the element.